service/history/workflow/cache/cache.go

```diff
@@ -475,4 +475,4 @@ func makeCacheKey(
 		WorkflowKey: definition.NewWorkflowKey(namespaceID.String(), execution.GetWorkflowId(), execution.GetRunId()),
 		ShardUUID:   shardContext.GetOwner(),
 	}
-}
```

Comment:
running `make goimports; make lint` was complaining by requiring me to include this.